templates/Features/ShareTarget.Prism/ViewModels/ShareTargetFeatureViewModel.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Param_ItemNamespace.ViewModels
 {
-    public class wts.ItemNameViewModel : BindableBase
+    public class ShareTargetFeatureViewModel : BindableBase
```

Comment:
This seems like a bad name. Why switch from the one based on the template item name?

Reply:
The sourceName in the template file is ShareTargetFeature, so that would be replaced at generation time instead of wts.ItemName.